Pushing prow pod utils images using ko

chaodaiG opened the pull request:
Local testing looks promising.

/hold
Depends on: https://github.com/kubernetes/test-infra/pull/25154

/cc @BenTheElder @cjwagner

cjwagner commented on prow/prow-images.sh:
```diff
@@ -21,7 +21,38 @@ REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+    for arch in "${ALL_ARCHES[@]}"; do
+      tags+=("${arch}")
+      for base in "${TAG_SET[@]}"; do
+        tags+=("${arch}-${base}")
```
This is backwards it should be base-arch.

cjwagner commented on prow/prow-images.sh:
```diff
@@ -21,7 +21,38 @@ REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+# push or local tar?
+PUSH="${PUSH:-false}"
+
+# takes comma separated list of arch
```
This isn't accurate.

cjwagner commented on prow/.prow-images:
```diff
@@ -1,2 +1,5 @@
-prow/cmd/deck
```
This looks like an unintentional removal?

chaodaiG commented:
#25157  contains very similar changes in `prow/prow-images.sh`, and advanced further than this PR, addressed reviews over there, please review that one instead

/hold

chaodaiG commented:
/close
will open a new PR
